mmv1/templates/terraform/examples/dlp_discovery_config_cloud_storage.tf.erb

```diff
@@ -0,0 +1,81 @@
+resource "google_data_loss_prevention_discovery_config" "<%= ctx[:primary_resource_id] %>" {
+	parent = "projects/<%= ctx[:test_env_vars]['project'] %>/locations/us"
```

Comment:
Nit: Spacing is a bit off here. This example will show up at the top of the documentation page for this resource, so inconsistencies in the formatting will be more visible

Reply:
Fixed with another commit and squashed into previous.